src/WooCommerce.php

```diff
@@ -740,6 +740,52 @@ public static function woocommerce_save_product_variation($post_id, $loop) {
      update_post_meta($variation_id, self::FIELD_MARKETING_FOCUS, $marketingFocusFieldValue);
   }
 
+  /**
+   * Allows adding multiple products to the cart.
+   *
+   * See https://bit.ly/3JfcIET
+   *
+   * @implements wp_loaded
+   */
+  public static function add_multiple_products_to_cart() {
+    if (
+      !isset($_REQUEST['add-to-cart']) ||
+      strpos($_REQUEST['add-to-cart'], ',') === false
+    ) {
+      return;
+    }
+
+    remove_action('wp_loaded', array('WC_Form_Handler', 'add_to_cart_action'), 20);
+
+    $product_ids = explode(',', $_REQUEST['add-to-cart']);
+    if (!$product_ids) {
+      return;
+    }
```

Comment:
@juanlopez4691 — I think explode would always return at least 1 id, this check is not needed here in my opinion
```suggestion
```

Reply:
I prefer to keep it, just in case no product ID at all is added to the URL. Just to be defensive.